src/test/java/com/notably/model/block/BlockTreeTest.java

```diff
@@ -0,0 +1,110 @@
+package com.notably.model.block;
+
+import static com.notably.testutil.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.notably.commons.core.path.AbsolutePath;
+import com.notably.commons.core.path.exceptions.InvalidPathException;
+import com.notably.model.block.exceptions.CannotModifyRootException;
+import com.notably.model.block.exceptions.DuplicateBlockException;
+import com.notably.model.block.exceptions.NoSuchBlockException;
+
+public class BlockTreeTest {
+    private static AbsolutePath toRoot;
+    private static AbsolutePath toCs2103;
+    private static AbsolutePath toCs2103Week1;
+    private static AbsolutePath toCs2103Week1Lecture;
+
+    private BlockTree blockTree = new BlockTreeImpl();
```

Comment:
Hi @firzanarmani , sorry for nitpicking but just curious hehe! Is there a reason why you initialise `blockTree` here and not inside the `setUpTestTree` method?

Reply:
Valid point! I agree and moved it in to `setUpTestTree`!